include/Model/Hint.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef HEXADOKU_HINT_HPP
+#define HEXADOKU_HINT_HPP
+
+class Hint {
+public:
+    Hint();
+
+    Hint(int column, int row, int value);
+
+    void SetColumn(int column);
+
+    int GetColumn();
+
+    void SetRow(int row);
```

Comment:
bedzie to potrzebne, kiedy wrzucamy to przy inicie?

Reply:
Niewykluczone